packages/cli/src/lib/commands/collect/options/openReport.ts

```diff
@@ -1,19 +1,15 @@
-import { argv } from 'yargs';
-import { Param } from './openReport.model';
-import { ArgvOption } from '../../../core/yargs/types';
+import { argv, Options } from 'yargs';
 import { getEnvPreset } from '../../../pre-set';
 
-export const param: Param = {
-  openReport: {
-    alias: 'e',
-    type: 'boolean',
-    description: 'Opens browser automatically after the user-flow is collected. (true by default)',
-    default: getEnvPreset().openReport as boolean,
-    requiresArg: true
-  }
-};
+export const openReport = {
+  alias: 'e',
+  type: 'boolean',
+  description: 'Opens browser automatically after the user-flow is collected. (true by default)',
+  default: getEnvPreset().openReport as boolean,
+  requiresArg: true
+} satisfies Options;
 
 export function get(): boolean {
-  const { openReport } = argv as any as ArgvOption<Param>;
+  const { openReport } = argv as any as { openReport: boolean };
```

Comment:
The previous typing was incorrect! So is this one. 

The Solution is to not use this pattern!